diff --git a/src/ShopifyApp/Exceptions/MissingShopDomainException.php b/src/ShopifyApp/Exceptions/MissingShopDomainException.php
index baa7c083798a49403f448feb64e5a7ec97447f4f..2d8017fb28bec3b56865d66dc681a454a6bb0fba 100644
--- a/src/ShopifyApp/Exceptions/MissingShopDomainException.php
+++ b/src/ShopifyApp/Exceptions/MissingShopDomainException.php
@@ -4,5 +4,4 @@ namespace OhMyBrew\ShopifyApp\Exceptions;
 
 class MissingShopDomainException extends \Exception
 {
-    
-}
\ No newline at end of file
+}
diff --git a/src/ShopifyApp/Exceptions/SignatureVerificationException.php b/src/ShopifyApp/Exceptions/SignatureVerificationException.php
index 74efb546f7f95f0a58e8ab639cbdac10bef7fff6..0b7e5b0d69740ee70f620ec16d0b7da0a9fed72b 100644
--- a/src/ShopifyApp/Exceptions/SignatureVerificationException.php
+++ b/src/ShopifyApp/Exceptions/SignatureVerificationException.php
@@ -4,5 +4,4 @@ namespace OhMyBrew\ShopifyApp\Exceptions;
 
 class SignatureVerificationException extends \Exception
 {
-    
-}
\ No newline at end of file
+}
diff --git a/src/ShopifyApp/Middleware/AuthShop.php b/src/ShopifyApp/Middleware/AuthShop.php
index df293b23fc1c89051aacf58fe9083d912472ba50..316a963d10288dc9d40ce47d64dd1bae1120b4e8 100644
--- a/src/ShopifyApp/Middleware/AuthShop.php
+++ b/src/ShopifyApp/Middleware/AuthShop.php
@@ -7,13 +7,12 @@ use Exception;
 use Illuminate\Http\Request;
 use Illuminate\Http\Response;
 use Illuminate\Support\Facades\Config;
-use Illuminate\Support\Facades\Session;
 use Illuminate\Support\Facades\Redirect;
-use OhMyBrew\ShopifyApp\Facades\ShopifyApp;
-use OhMyBrew\ShopifyApp\Services\ShopSession;
-use OhMyBrew\ShopifyApp\Services\AuthShopHandler;
+use Illuminate\Support\Facades\Session;
 use OhMyBrew\ShopifyApp\Exceptions\MissingShopDomainException;
 use OhMyBrew\ShopifyApp\Exceptions\SignatureVerificationException;
+use OhMyBrew\ShopifyApp\Facades\ShopifyApp;
+use OhMyBrew\ShopifyApp\Services\ShopSession;
 
 /**
  * Response for ensuring an authenticated shop.
